### ISSUE TITLE
[CTW/16v16] 803

### ISSUE THREAD

**Issue by minuyo** [OvercastCommunity/CommunityMaps#885]
### Checklist

- [X] I have [pruned](https://pgm.dev/docs/guides/preparing/pruning-chunks) the map. (If you used OCC Mapmaker and downloaded your map, it has been pruned for you.)
- [X] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the [README](https://github.com/OvercastCommunity/CommunityMaps#readme).
- [X] I have read and understood the  [map submission guidelines](https://oc.tc/docs/submissions).
- [X] I have created an XML file.
- [X] I have created a map image.
- [X] I have uploaded the map zip file to a file sharing service.
- [X] The map has been tested locally to make sure it works.

### Map Name

803

### Gamemode & Map Description

It's a simple CTW map. In the inventory is a splash potion of harming 1, which should be a small game changer, if it turns out to be too strong, we can remove it. 

### Team Sizes

16v16

### Screenshots

https://imgur.com/a/mzSUyCv

### XML

https://gist.github.com/minuyo/0ade7f885edb1c22350c51fa9980da7a

### Map Image

![2024-06-29_22 58 59](https://github.com/OvercastCommunity/CommunityMaps/assets/151182727/698b4f56-d40d-4868-878b-172ab52804a9)

### Map Download

[803.zip](https://github.com/user-attachments/files/16042206/803.zip)

**Comment by technodono** [comment]
Besides the wool lane Is the map Just a straight line? I can't tell cause there's no overhead preview but my concern with that is with the map essentially becoming a "brick wall" style of gameplay where players are just battling it at mid to try break through the hold.

This is just my initial assumptions though, you don't have to follow through with these feedback.

**Comment by technodono** [comment]
> > Besides the wool lane Is the map Just a straight line? I can't tell cause there's no overhead preview but my concern with that is with the map essentially becoming a "brick wall" style of gameplay where players are just battling it at mid to try break through the hold.
> > This is just my initial assumptions though, you don't have to follow through with this feedback.
> 
> ![image](https://private-user-images.githubusercontent.com/84674701/344676666-3f8b28b2-6909-4701-a1ca-61bbc2a72ece.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4MzgwMzcsIm5iZiI6MTcxOTgzNzczNywicGF0aCI6Ii84NDY3NDcwMS8zNDQ2NzY2NjYtM2Y4YjI4YjItNjkwOS00NzAxLWExY2EtNjFiYmMyYTcyZWNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAxVDEyNDIxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkMGMxYzJlOTBlNDIyNzU3MTE1ZjkyNThiZDc0N2M2ZTRkMTM5ODc3ZWRhMmE1N2Y0YjdiNWU3ZjUxYjNmMWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UgKI9bNpwQUPkvRDiiXwj2ZtTPbb3EqUIL7dGW80T7s)

Yeah, my concern still stands. There's no flanks or other routes for players to use to help them get into/past enemy lines or pathing. so the whole map would essentially end up either being who has the better ground or sky push.

**Comment by technodono** [comment]
I am assuming that goes directly to the woolroom correct? Don't get me wrong it definitely helps but the long map length compounded with the mostly predominant single lane will lead to unfun matches. The same general linear design has been seen in similar rejected submission such as the one below.
https://github.com/OvercastCommunity/CommunityMaps/issues/506#issue-1393647494

My suggestions for improvements (not sure if other mapdevs have anything to add) would be to move the islands between mid to shortest and add some sort of secondary route to the side of the road which would enable some dual lane pressure alongside the existing pipe flank route

**Comment by minuyo** [comment]
okay i have an idea what I can change at the map, I will do it when I have the time

**Comment by minuyo** [comment]
Changed some things at the map, basically I just added another island at the side leading to the wool room 

Map: 
[803.zip](https://github.com/user-attachments/files/17103887/803.zip)


XML:
[map.xml.zip](https://github.com/user-attachments/files/17103900/map.xml.zip)


![2024-09-23_22 33 55](https://github.com/user-attachments/assets/9e95cc97-791e-4118-b617-bebc9fcd6e9f)

**Comment by calcastor** [comment]
In its current state this map is significantly too long and narrow to be appropriate for the map pool. The wool room is also much too close to spawn.

**Comment by minuyo** [comment]
Actually the idea of the map is that it is narrow, so you need to carry the wool for a long distance, its inspired by the one race for victory map where you need to get over the lava pool in the middle.

Should I cut it smaller??

**Comment by Pablete1234** [comment]
Since you compared your map to race for victory, let's run some numbers:

803:
 - From where you spawn (150,25) to the enemy woolroom (-110,-67), about 260,92, or **352** blocks in taxicab distance
 - Of that distance, 52 blocks need to be bridged over (4 gaps of 13 )
 - The lane gets as narrow as 16 blocks
 - Has weird turns that make it harder to bridge/skybridge on as its easier to get knocked off

RFV:
 - From where you spawn (0,110) to the enemy woolroom (-26,-95), about 26,205, or **230** blocks in taxicab distance
 - Of that distance, only 14 need to be bridged (can be as little as 6 blocks if you go down to the lava, or, a single water bucket and no bridging at all)
 - The lane is a constant 24 blocks wide
 - A skybridge that goes straight across is possible

The answer is yes, this map is absolutely too long for how wide it is.

Either A, it is made significantly wider, and upped to play with significantly more players, or B, it is made significantly shorter and keeps the playersize

**Comment by eliminate123** [comment]
Thank you for your submission. Im confident in this map, and I've put it in the pool.
